File: 0001 Code Chef/0018 Saving Taxes/implementation/0006 php/saving_taxes.php
<?php
    $test  = (int) fgets(STDIN);
    while($test > 0){
        $string_input = fgets(STDIN);
        $string_array = explode(" ",$string_input);
        $x = (int) $string_array[0];
        $y = (int) $string_array[1];
        if($x > $y){
            $saving = $x - $y;
            echo "$saving\n";
        }else{
            echo "0\n";
        }
        $test = $test - 1;
    }
?>

File: 0001 Code Chef/0018 Saving Taxes/implementation/0011 lua/saving_taxes.lua
test = tonumber(io.read())

while(test > 0) do
    string_input = io.read()
    string_array = {}
    index = 1
    for i in string.gmatch(string_input,"%S+") do
        string_array[index] = i
        index = index + 1
    end

    x = tonumber(string_array[1])
    y = tonumber(string_array[2])

    if(x > y) then
        print(x-y)
    else
        print(0)
    end

    test = test - 1
end